internal/etl/transform/extend-values.go

```diff
@@ -0,0 +1,120 @@
+package transform
```

Comment:
de naam van de file, zouden we dat niet beter `substitutions.go` kunnen noemen?

Overigens zou ik het sowieso `extend_values` ipv `extend-values` noemen aangezien underscores Go convention is.

Reply:
Ik verwacht sommige func te kunnen hergebruiken voor de synonyms, vandaar dat ik een algemenere naar heb gekozen. Zal er extend_values.go van maken.